extensions/vscode/src/multiStepInputs/selectNewOrExistingConfig.ts

```diff
@@ -126,7 +126,9 @@ export async function selectNewOrExistingConfig(
       configFileListItems.push({
         iconPath: new ThemeIcon("plus"),
         label: createNewConfigurationLabel,
-        detail: "(or pick one of the existing deployments below)",
+        detail: configurations.length
+          ? "(or pick one of the existing configurations below)"
```

Comment:
This also corrects the text from `deployments` to `configurations` since configurations are listed. For context this is the multiStepInput when clicking "Select Active Configuration for Deployment" in the three dot context menu.

There shouldn't be a case where this detail is `undefined` since when a Deployment is missing its Configuration file and we click this context menu it immediately starts configuration creation. There isn't much harm in including the option though.